vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -401,13 +387,12 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
 
   /**
    * Callback to update the address data after the user chooses a suggested address.
-   *
-   * @param address the data object returned by the AddressComplete Retrieve API.
+   * @param address the data object returned by the AddressComplete Retrieve API
    */
   private addressCompletePopulate (address: object): void {
     const newAddressLocal: object = {}
 
-    newAddressLocal['streetAddress'] = address['Line1']
+    newAddressLocal['streetAddress'] = address['Line1'] || 'N/A'
```

Comment:
While testing, I noticed some address (eg, in Algeria) don't have any LineX data, so I added this fallback so that an address filled in my AddressComplete is valid (since street address is typically a required field).